src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,58 @@
+package duke.task;
+
+
+/**
+ * a class that represent todo task
+ */
+public class Todo extends Task {
+    private String name;
+    private boolean isDone;
+
+    /**
+     * constructor
+     * @param name
+     * @param isDone
+     */
+    public Todo(String name, boolean isDone) {
+        super();
+        this.name = name;
+        this.isDone = isDone;
+    }
+
+    /**
+     * search whether keyword exist
+     * @param keyword
+     * @return
+     */
+    public boolean searchKeyword(String keyword) {
+        String[] s = name.split(" ");
+        for (String ss : s) {
+            if (ss.equals(keyword)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
```

Comment:
The function name `searchKeyword` should be changed to `hasKeyword` to make it more like a boolean. The way you match keywords is very concise and clear. Would you consider making it case-insensitive?